src/mpi/request/waitany.c

```diff
@@ -71,14 +134,11 @@ int MPI_Waitany(int count, MPI_Request array_of_requests[], int *indx,
     static const char FCNAME[] = "MPI_Waitany";
     MPIR_Request * request_ptr_array[MPIR_REQUEST_PTR_ARRAY_SIZE];
     MPIR_Request ** request_ptrs = request_ptr_array;
-    MPID_Progress_state progress_state;
     int i;
-    int n_inactive;
     int active_flag;
-    int init_req_array;
-    int found_nonnull_req;
-    int last_disabled_anysource = -1;
+    int found_nonnull_req = FALSE;
     int mpi_errno = MPI_SUCCESS;
+    int off = 0;
```

Comment:
Is this used?

Reply:
Nope, left over from the old code, will remove -- thanks!